frontend/src/components/Image/styled.tsx

```diff
@@ -1,29 +1,14 @@
 import styled from 'styled-components';
 
-import { SIZE } from 'constants/sizes';
-
-interface Size {
-  size?: string;
-}
-
-export const ImageContainer = styled('div')<Size>(({ size }) => ({
-  overflow: 'hidden',
-  borderRadius: '10px',
-  textAlign: 'center',
-  margin: '5px',
+export const ImageContainer = styled('div')({
+  margin: '0px',
   float: 'left',
-  width: '150px',
-  height: '100px',
-  ...(size === SIZE.large && {
-    width: '450px',
-    height: '300px',
-  }),
-  ...(size === SIZE.medium && {
-    width: '300px',
-    height: '200px',
-  }),
-  ...(size === SIZE.small && {
-    width: '150px',
-    height: '100px',
-  }),
-}));
+  width: '100%',
+  height: '100%',
```

Comment:
Do we need a container with width 100%?

Reply:
Wep, because this counter is 'conductor' between couple other components. Lets see we have: ImageWrapper > ImageContainer > CurrentImage. Image wrapper take in yourself `width: some px , height: some px`, ImageContainer have to have accurate width and height. For this  `  width: '100%',
  height: '100%',`

Reply:
And why do we need ImageContainer?

Reply:
I mean this `const Image: React.FC<Props> = ({ imageUrl }) => (
  <ImageContainer>
    <CurrentImage src={imageUrl || imageDefault} />
  </ImageContainer>
);` its our component from previously tasks